app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -75,7 +76,21 @@ def service_type
     end
 
     def redirect_to_data_review_if_intake_authenticated
-      redirect_to StateFile::Questions::DataReviewController.to_path_helper(us_state: params[:us_state]) if current_state_file_az_intake.present? || current_state_file_ny_intake.present?
+      intake = current_state_file_az_intake || current_state_file_ny_intake
+      if intake.present?
+        if intake.hashed_ssn.present?
+          redirect_to StateFile::Questions::DataReviewController.to_path_helper(us_state: params[:us_state])
+        else
+          redirect_to StateFile::Questions::TermsAndConditionsController.to_path_helper(us_state: params[:us_state])
+        end
+      end
+    end
+
+    def session_sign_in
+      intake = @records.first
+      sign_in intake
+      session[:state_file_intake] = intake.to_global_id
+      session.delete(:after_state_file_intake_login_path)
```

Comment:
i assume we're trying to redirect to after_state_file_intake_login_path here. does returning the value do that?